services/submission/src/test/resources/application.yaml

```diff
@@ -33,6 +33,8 @@ services:
       path: /version/v1/tan/verify
     monitoring:
       batch-size: 5
+    diagnosis-key:
+      rolling-period: 144
```

Comment:
Does this configuration represent the maximum rolling period? If yes, it should be evident from the name